### DISCUSSION TITLE
QuartzServer

### DISCUSSION THREAD

**Discussion by codeputer** [quartznet/quartznet#392] [General] [answered]
I'm a newbie on Github, as well as Quartznet, so please excuse me posting this as an issue as this is not so much an issue, but more of question.  I'm reviewing the class QuartzServer, and the integration with TopShelf.... is there any examples of how I would use this class?  Is this a question more suitable for StackOverFlow?

I've created my own service using TopShelf, and now I want to start the scheduler, and attach a job to it....

Regards

**Comment by lahma** [comment, marked as answer]
Probably easiest is to open the source of the server in Visual Studio and run it to see how it works. It's more of a template how to get started and it should just work by running from Visual Studio, there you can change parameters etc, via job configuration XML.
